web/client/codechecker_client/cmd/store.py

```diff
@@ -894,7 +894,18 @@ def main(args):
                                                  port,
                                                  product_name=product_name)
 
-    zip_file_handle, zip_file = tempfile.mkstemp('.zip')
+    temp_dir = "/".join([args.input[0], "tmp"])
```

Comment:
Please, use the python pathlib for building file paths so the implementation is more platform independent.